lib/hyacinth/config.rb

```diff
@@ -58,5 +58,22 @@ def self.external_identifier_adapter
     def self.term_search_adapter
       @term_search_adapter ||= Hyacinth::Adapters.create_from_config('Hyacinth::Adapters::TermSearchAdapter', HYACINTH[:term_search_adapter])
     end
+
+    def self.default_lang_value
+      @default_lang_value ||= ::Language::Tag.for(LANG[:default_lang_value])
+    end
+
+    def self.allowed_lang_value?(tag_value)
+      ::Language::Tag.for(tag_value)
+    rescue
+      false
+    end
+
+    def self.load_default_subtags!
+      Rails.application.config_for(:lang).fetch(:default_lang_subtags, {}).each do |subtag, attributes|
```

Comment:
```suggestion
      LANG.fetch(:default_lang_subtags, {}).each do |subtag, attributes|
```
Okay to use LANG?

Reply:
It creates a dependency on initializer execution that I think might not be desirable here - I'd want to revisit and make sure tests run, etc. Can we ticket as a cleanup task?

Reply:
Later cleanup task is fine, but aren't the initializers always going to be loaded before you use the Hyacinth::Config object?  You're already using LANG a couple of methods up in the `default_lang_value`.

Reply:
This method gets called in contexts that `default_lang_value` does not (eg specs)

Reply:
Ah, I see what you mean.  Okay, I take back my original suggestion.  Here's what I'd recommend instead:  I don't think this method belongs in Hyacinth::Config.  The Hyacinth::Config class isn't supposed to actually _do_ things.  It's just supposed to _provide_ things.  Can we move this into a different module, along with the `allowed_lang_value?` method?  Maybe `Hyacinth::Language`?

Reply:
These two methods have been moved to `Hyacinth::Language`